supabase/migrations/20260503000000_exercise_catalog.sql

```
@@ -0,0 +1,130 @@
+    ON exercise_catalog (LOWER(name));
+CREATE INDEX IF NOT EXISTS idx_exercise_catalog_muscle
+    ON exercise_catalog (muscle_group);
+CREATE INDEX IF NOT EXISTS idx_exercise_catalog_movement
+    ON exercise_catalog (movement);
+CREATE INDEX IF NOT EXISTS idx_exercise_catalog_custom
+    ON exercise_catalog (user_id) WHERE is_custom = TRUE;
+
+-- 3. RLS
+ALTER TABLE exercise_catalog ENABLE ROW LEVEL SECURITY;
+
+CREATE POLICY "Library exercises visible to all authenticated users"
```

SQLFluff (4.1.0)

[error] Do not use special characters in identifiers.

(RF05)